qa/rpc-tests/test.py

```diff
@@ -0,0 +1,235 @@
+# /************************************************************************
+#  File: test.py
+#  Author: mdr0id
+#  Date: 1/7/2018
+#  Description:  Used to run RPC test per platform
+#
+#  Usage: python test.py
+#
+#  Notes:
+#
+#  Known bugs/missing features:
+#  1. Win / Lin / Mac signal handler (need to go back and check/implement) 
+#
+# ************************************************************************/
+
+import os
+import psutil
+import platform
+import time
+import logging
+import argparse
+from subprocess import Popen, PIPE
+
+logging.basicConfig(format='%(asctime)s - PID:%(process)d - %(levelname)s: %(message)s', level=logging.INFO)
+
+#get system information to setup according work directory for RPCs
+HOST_OS = platform.system()
+
+#get host platform path for current user
+WORK_DIR = os.path.expanduser('~')
```

Comment:
Do we need this to work when the user's home directory contains non-ASCII characters? If we do, see https://github.com/tahoe-lafs/tahoe-lafs/blob/b31b78e6cf7d08da0c5e579083bc36ee27ca7685/src/allmydata/util/fileutil.py#L383-L443

Reply:
Ah, good catch! Testing the non-ASCII issues was fun!